contrib/docker/README.md

```diff
@@ -101,105 +90,322 @@ value.
    $ docker logs "${DCRD_CONTAINER_NAME}"
    ```
 
+5. Don't forget to configure any host and network firewalls to allow access to
+   the peer-to-peer port and potentially setup port forwarding if the host is
+   using Network Address Translation (NAT) if you want to allow inbound
+   connections to contribute to network decentralization.
+
+### Querying `dcrd` with `dcrctl` Inside the Running Container
+
+```sh
+$ docker exec "${DCRD_CONTAINER_NAME}" dcrctl getblockchaininfo
+```
+
+### Showing available `dcrctl` Commands Inside the Running Container
+
+```sh
+$ docker exec "${DCRD_CONTAINER_NAME}" dcrctl -l
+```
+
+**TIP:** The `dcrctl` utility interfaces with both `dcrd` and `dcrwallet`.
+Since the container only provides `dcrd`, which acts as a chain server, only the
+commands listed under "Chain Server Commands" are available.
+
 ### Starting and Stopping the Container
 
-   ```sh
-   $ docker stop "${DCRD_CONTAINER_NAME}"
-   $ docker start "${DCRD_CONTAINER_NAME}"
-   ```
+```sh
+$ docker stop "${DCRD_CONTAINER_NAME}"
```

Comment:
[docker stop](https://docs.docker.com/engine/reference/commandline/stop/) will send a `SIGTERM`, and if the container hasn't stopped in 10 seconds it sends a `SIGKILL`. Might want to extend that timeout, e.g. for 60 seconds:

```sh
$ docker stop -t 60 "${DCRD_CONTAINER_NAME}"
```